### PR TITLE
added oneliners flag

### PR DESCRIPTION
**Pull request by rgeronimi**
Added a "oneliners" flag to follow the tradition of most Go tools, i.e., output one line per case

### CODE REVIEW

File: unconvert.go [hunk 1]
```diff
@@ -157,6 +162,7 @@ var (
 	flagAll        = flag.Bool("all", false, "type check all GOOS and GOARCH combinations")
 	flagApply      = flag.Bool("apply", false, "apply edits to source files")
 	flagCPUProfile = flag.String("cpuprofile", "", "write CPU profile to file")
+	flagOneLiners = flag.Bool("oneliners", false, "outputs 1 line per case")
```
Comment (shurcooL) [inline unconvert.go]: This doesn't look `gofmt`ed.

**Comment by shurcooL** [comment]
> tradition of most Go tools

Can you post a few examples? This is the first time I see such a flag.

**Comment by rgeronimi** [comment]
Hello, sorry for the gofmt, I will add it.

Here is a list of the tools I use, all of them excepted one (dupl) output 1 line per case by default. The exceptional one, dupl, has a specific "plumbing" flag to output 1 line per case. I invented the "one-liner" name, please feel free to choose whatever name sounds right.

gofmt
go tool vet
golint
errcheck
interfacer
copyrighter
align check
structcheck
varcheck
ineffassign
dupl
misspell

**Comment by shurcooL** [comment]
Ah, the "tradition" you were referring to is to output a single line per result, it wasn't about having a "oneliners" flag. I misunderstood.

**Comment by dominikh** [comment]
One term I see used for this from time to time is "quickfix", because of the vim feature and the format associated with it. Clang uses -fno-caret-diagnostics I believe, but that's obviously not a good choice here.

**Comment by mdempsky** [comment]
Thanks for the feedback.  I've changed the default behavior better match other tools, and the extra line/caret output is behind a `-v` flag now.

**Comment by rgeronimi** [comment]
great thanks

**Pull request closed without merging**